QUEST.tsv

```diff
@@ -45,251 +45,251 @@ QUEST_20150317_000044	Items to be deleted
 QUEST_20150317_000045	Select GenType
```

Comment:
The line itself isn't to be deleted. The line just states "Items to be deleted."